packages/protocol/truffle.js

```diff
@@ -19,12 +19,14 @@ const ALFAJORES_FROM = '0x456f41406B32c45D59E539e4BBA3D7898c3584dA'
 const PILOT_FROM = '0x387bCb16Bfcd37AccEcF5c9eB2938E30d3aB8BF2'
 const PILOTSTAGING_FROM = '0x545DEBe3030B570731EDab192640804AC8Cf65CA'
 
+const gasLimit = 8000000
```

Comment:
do we need to increase the gasLimit? 
@m-chrzan ??

Reply:
Very likely. Smart contracts have increased in bytecode so might need to pay a higher fee for deploying them.